crates/matrix-sdk-ui/src/timeline/event_handler.rs

```diff
@@ -846,11 +866,7 @@ impl<'a, 'o> TimelineEventHandler<'a, 'o> {
     }
 
     /// Add a new event item in the timeline.
-    fn add(&mut self, should_add: bool, content: TimelineItemContent) {
-        if !should_add {
-            return;
-        }
-
+    fn add_new_item(&mut self, content: TimelineItemContent) {
```

Comment:
Can we call this `add_item`? The `new` part sounds just a bit redundant, since we're `add`ing something…

Reply:
:+1: